_components/table/table.md

```diff
@@ -41,20 +43,23 @@ tags:
   - borderless
   - scrollable
   - scrolling
+  - sticky
 title: Table
 type: component
 variants:
-  - variant: "`usa-table--borderless`"
+  - variant: "`.usa-table--borderless`"
     description: Removes the outer table borders, retaining only a single bottom border on each row. Best for tables with more text than numbers.
-  - variant: "`usa-table--compact`"
+  - variant: "`.usa-table--compact`"
     description: Reduces the row height and vertical spacing to display more table rows within a limited space. Should only be used with dense, numerical data, not text content. Pairs well with scrollable and striped variants, but is not suitable for use with stacked variants.
   - variant: "`.usa-table-container--scrollable > .usa-table`"
     description: Add the `usa-table-container--scrollable` class to a container around any `usa-table` to apply a horizontal scrollbar if the columns exceed the available width. Ideal for dense tables with many columns.
-  - variant: "`usa-table--stacked`"
-    description: Stacks the table cells on narrow screens. Ideal for tables that contain more text information than numerical data. If you use this variant, you must ensure there is a data-label attribute on each cell of the table that matches the column header.
-  - variant: "`usa-table--stacked-header`"
+  - variant: "`.usa-table--stacked`"
+    description: Stacks the table cells on narrow screens. Ideal for tables that contain more text information than numerical data. If you use this variant, you must ensure there is a `data-label` attribute on each cell of the table that matches the column header.
+  - variant: "`.usa-table--stacked-header`"
     description: Stacks the table cells on narrow screens and visually promotes the first cell of every row into a “header” for that group. Preferred for directories and other lists where the first cell of every row is a name. If you use this variant, you must ensure there is a data-label attribute on each cell of the table that matches the column header.
-  - variant: "`usa-table--striped`"
+  - variant: "`.usa-table--sticky-header`"
+    description: Allows the table header element to stick to the top of the page as the user scrolls. Preferred for long tables with many rows. This variant is not compatible with the scrollable and stacked table variants.
+  - variant: "`.usa-table--striped`"
```

Comment:
## Optional: PR description update

Since we're adding the guidance in this PR, it might be nice to include the nice tables you included in https://github.com/uswds/uswds/pull/5420 in this PR description. I found myself viewing https://github.com/uswds/uswds/pull/5420 to compare 

### New variant

| Variant name              | Description | 
| ---------------------------- | :-------------- |
| `.usa-table--sticky-header` |   Allows the table header element to stick to the top of the page as the user scrolls. Preferred for long tables with many rows. This variant is not compatible with the scrollable and stacked table variants. |   

### New settings

| Setting name              | Description | 
| ---------------------------- | :-------------- |
| `$theme-table-background-color` |   Defines the table background color. A value of `default` uses the body background color. |   
| `$theme-table-sticky-top-offset` |  Defines the top offset for sticky headers. |

Reply:
Good note - added these tables to the PR description for easier reference